community/communities.md

```diff
@@ -89,3 +89,21 @@ Link: [https://stackoverflow.com/questions/tagged/machine-translation](https://s
 The **Machine-Translation** tag on Quora gathers user questions and answers related to machine translation, including its development and performance.
 
 Link: [https://www.quora.com/topic/Machine-Translation/](https://www.quora.com/topic/Machine-Translation/)
+
+
+## Communities
+
+### MaCoCu
+
+**MaCoCu** focuses on collecting monolingual and parallel data from the Internet, specially for under-resourced languages and DSI-specific data.
+
+This action takes place from June 2021 to June 2023 (2 years) and it is co-financed by the European Union.
+
+Link: [https://macocu.eu/](https://macocu.eu/)
```

Comment:
I'd avoid *Link:* since it's obvious.  Can make it a single bullet point, or a line right under the header.

Also remove the *https://* from the visible part.